topics/babel/index.md

```diff
@@ -0,0 +1,12 @@
+---
+created_by: Sebastian McKenzie, James Kyle, Henry Zhu, Logan Smyth, Daniel Tschinder
+display_name: Babel
+github_url: https://github.com/babel
+logo: babel.png
+released: Sep 28, 2014
+short_description: 🐠 Babel is a compiler for writing next generation JavaScript.
+topic: babel
+url: https://babeljs.io/
+---
+
+🐠 Babel is a compiler created by Sebastian McKenzie in 2014 originally created to convert ES6 to ES5 (previously 6to5), but now is a toolchain to allow developers to write any next generation JavaScript and testing ground for proposals from TC39, the technical committee that specifies ECMAScript. Babel can also convert JSX syntax and strip out type annotations. Babel is built out of plugins. Compose your own transformation pipeline using existing plugins or write your own.
```

Comment:
Just my 2cents:

> 🐠 Babel is a compiler for writing next generation JavaScript.

🐠 Babel is a compiler for writing next generation JavaScript, today.

> Babel can also convert JSX syntax and strip out type annotations

Babel can also convert JSX syntax and strip out type annotations from both Flowtype and TypeScript

> Compose your own transformation pipeline using existing plugins or write your own

Compose your own transformation pipeline using the plugins written by the community or write your own

Reply:
Updated :+1: